packages/api/src/routers/api/team.ts

```diff
@@ -144,4 +148,43 @@ router.patch('/apiKey', async (req, res, next) => {
   }
 });
 
+router.get('/tags', async (req, res, next) => {
+  try {
+    const teamId = req.user?.team;
+    if (teamId == null) {
+      throw new Error(`User ${req.user?._id} not associated with a team`);
+    }
+    const simpleCache = new SimpleCache(
+      `tags:${teamId}`,
+      ms('10m'),
+      async () => {
+        const _tags: string[] = [];
+        const dashboards = await Dashboard.find(
```

Comment:
We've gotten a bit loose with our standards recently, but our preference is actually that mongoose code lives under a controller, maybe under the `controllers/team.ts` in this case - so that we can encapsulate data fetching separate from router logic. I know that several of our routes don't do this but we've recently talked about trying to get back on track again :)

Reply:
updated